x-pack/plugins/security_solution/cypress/e2e/detection_rules/alerts_table_threat_indicator.cy.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { CustomRule } from '../../objects/rule';
+import { getNewThreatIndicatorRule } from '../../objects/rule';
+import {
+  ALERTS_COUNT,
+  ALERT_TABLE_ADDITIONAL_CONTROLS,
+  ALERT_TABLE_SHOW_THREAT_INDICATOR,
+} from '../../screens/alerts';
+import { goToRuleDetails } from '../../tasks/alerts_detection_rules';
+import { cleanKibana } from '../../tasks/common';
+import {
+  createAndEnableRule,
+  fillAboutRuleAndContinue,
+  fillDefineIndicatorMatchRuleAndContinue,
+  fillScheduleRuleAndContinue,
+  selectIndicatorMatchType,
+  waitForAlertsToPopulate,
+} from '../../tasks/create_new_rule';
+import { esArchiverLoad, esArchiverUnload } from '../../tasks/es_archiver';
+import { login, visit, visitWithoutDateRange } from '../../tasks/login';
+import { ALERTS_URL, RULE_CREATION } from '../../urls/navigation';
+
+describe('Threat Indicator', () => {
+  before(() => {
+    cleanKibana();
+    esArchiverLoad('threat_indicator');
+    esArchiverLoad('suspicious_source_event');
+    login();
+    const rule = getNewThreatIndicatorRule();
+    visitWithoutDateRange(RULE_CREATION);
+    selectIndicatorMatchType();
+    fillDefineIndicatorMatchRuleAndContinue(rule);
+    fillAboutRuleAndContinue({ name: rule.name, description: rule.description } as CustomRule);
```

Comment:
Shouldn't it be possible to pass the whole rule object here?  Like,`fillAboutRuleAndContinue(rule)`.